### ISSUE TITLE
New release breaks module

### ISSUE THREAD

**Issue by phoqe** [oliviertassinari/react-swipeable-views#566]
- [x] I have searched the [issues](https://github.com/oliviertassinari/react-swipeable-views/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior

My app to compile.

## Current Behavior

Shows "Failed to compile" error screen. 

```
./node_modules/react-swipeable-views/lib/SwipeableViews.js
Module not found: Can't resolve 'dom-helpers/events/off' in '/Users/phoqe/WebstormProjects/react-material-ui-firebase/node_modules/react-swipeable-views/lib'
```

## Steps to Reproduce (for bugs)

Install the latest version and try to compile.

## Context

Running `yarn start`.

## Your Environment

| Tech                   | Version |
|------------------------|---------|
| react-swipeable-views  |0.13.4|
| React                  |16.12.0|
| platform               |macOS Catalina 10.15.2|
| etc                    |Safari Version 13.0.4 (15608.4.9.1.3)|

**Comment by KunwarCG** [comment]
I am facing the same issue on windows .

**Comment by phoqe** [comment]
@jeanpoelie Is there any update on this issue? Either rollback to 0.13.3 or create a new release fixing this issue ASAP as there are a lot of people depending on this library.

**Comment by theshashiverma** [comment]
I was also facing the same issue, but somehow I manage to solve this https://github.com/react-bootstrap/dom-helpers/issues/75

**Comment by jeanpoelie** [comment]
Unfortunatly i have need been able to do a successfull building release yet due to the project originally being made for mac building. My windows pc cannot run certain commands which are in the automated process.
Therefore i am debugging it on a mac pc but to no luck yet.

Unfortunatly i cannot rollback due to npm rules, there is no unpublish for files older than 3 days.

The above answer is a good workaround for now.

I hoped to get a release yesterday but i ran in too many errors with build commands.

**Comment by aaronwlee** [comment]
temporary solution:
npm install dom-helpers@3.2.1
or 
yarn add dom-helpers@3.2.1

**Comment by benitogf** [comment]
> Unfortunatly i have need been able to do a successfull building release yet due to the project originally being made for mac building. My windows pc cannot run certain commands which are in the automated process.
> Therefore i am debugging it on a mac pc but to no luck yet.
> 
> Unfortunatly i cannot rollback due to npm rules, there is no unpublish for files older than 3 days.
> 
> The above answer is a good workaround for now.
> 
> I hoped to get a release yesterday but i ran in too many errors with build commands.

Same here @jeanpoelie tried changing the build command (removing the && operator) but the build still fails 

```
lerna notice cli v3.20.2
lerna info Executing command in 3 packages: "babel --config-file babel.config.js src --out-dir lib --ignore test.js"
src\checkIndexBounds.js -> lib\checkIndexBounds.js
src\computeIndex.js -> lib\computeIndex.js
src\constant.js -> lib\constant.js
src\getDisplaySameSlide.js -> lib\getDisplaySameSlide.js
src\index.js -> lib\index.js
src\mod.js -> lib\mod.js
SyntaxError: src/autoPlay.js: Unexpected token (9:10)
   7 | export default function autoPlay(MyComponent) {
   8 |   class AutoPlay extends React.Component {
>  9 |     timer = null;
     |           ^
  10 | 
  11 |     constructor(props) {
  12 |       super(props);
```

unfortunately I'm not familiar with lerna, seems that the build is not taking the babel config (not using the plugins defined there) maybe a revert to 0.13.3 would be better for now?

**Comment by aaronwlee** [comment]
> Unfortunatly i have need been able to do a successfull building release yet due to the project originally being made for mac building. My windows pc cannot run certain commands which are in the automated process.
> Therefore i am debugging it on a mac pc but to no luck yet.
> 
> Unfortunatly i cannot rollback due to npm rules, there is no unpublish for files older than 3 days.
> 
> The above answer is a good workaround for now.
> 
> I hoped to get a release yesterday but i ran in too many errors with build commands.

Try to use wsl ,you may figure the windows pc command issues.

**Comment by jeanpoelie** [comment]
@benitogf thank you for that suggestion! I had not thought of trying that, i will do so today!


In the meantime i succesfully released 0.13.5 with the dom helper bug fix.

**Comment by yanickrochon** [comment]
`dom-helpers@3.2.1` ???

I the latest version is 5.1.3... So the latest update of this module requires an outdated dependency?
